### PR TITLE
Update: bump discovery-swarm-web to latest version

### PR DESCRIPTION
**Pull request by dpaez**
This PR updates `discovery-swarm-web` to its latest version.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Dat extends EventEmitter {
     const discoveryOpts = Object.assign({
       stream: (info) => this._replicate(info)
     }, this.opts)
-    this.swarm = new DiscoverySwarmWeb(discoveryOpts)
+    this.swarm = DiscoverySwarmWeb(discoveryOpts)
```
Comment (RangerMauve) [inline index.js:34]: Was there an error popping up when you had the new keyword here?

**Review by RangerMauve — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (dpaez) [reply index.js:34]: Yup, if we want to use the `new` keyword we have to do a named import. I think I can update discovery-swarm-web docs to highlight that.

Comment (RangerMauve) [reply index.js:34]: Interesting. Docs for that would be handy to have. :)

**Comment by RangerMauve** [comment]
Out in 8.0.2. Thanks again for the PR! :D

**Pull request merged**